python/python/glide/async_commands/core.py

```diff
@@ -132,20 +132,124 @@ class InfBound(Enum):
     NEG_INF = "-inf"
 
 
-class ScoreLimit:
+class LexInfBound(Enum):
```

Comment:
If this enum only differs from InfBound in the redis arg value ("+inf" vs "+"), but identical in meaning, lets simplify it for the user and have a single enum. you can do something like:
```python
class InfBound(Enum):
    POS_INF = {"default_arg": "+inf", "lex_arg": "+"}
    NEG_INF = {"default_arg": "-inf", "lex_arg": "-"}


self.start = start.value["default_arg"] if type(start) == InfBound else start.value